src/psyclone/psyir/symbols/routinesymbol.py

```diff
@@ -151,6 +152,22 @@ def is_elemental(self, value):
                             f"'{type(value).__name__}'")
         self._is_elemental = value
 
+    def copy(self):
+        '''Create and return a copy of this object. Any references to the
+        original will not be affected so the copy will not be referred
+        to by any other object.
+
+        :returns: A symbol object with the same properties as this \
```

Comment:
No need for line-continuation char (`\`).

Reply:
Removed